### ISSUE TITLE
No color appears with "fill-opacity" in my svg file

### ISSUE THREAD

**Issue by jbdujardin** [mchoe/SwiftSVG#76]
Hi!

When my *.svg file contains "fill-opacity" -> no color appears.

You can find my svg file attached (with 0.5 fill-opacity)

My code : 

``` swift
import SwiftSVG

let svgURL = URL(string: "https://jbdujardin.com/test.svg")!
let svgView = UIView(SVGURL: svgURL) { (svgLayer) in
      svgLayer.resizeToFit(CGRect(x: 0.0, y: 0.0, width: 320.0, height: 180.0))
}
svgView.frame = CGRect(x: 0.0, y: 0.0, width: 320.0, height: 180.0)
layoutView.addSubview(svgView)
```

Thanks for helping :)

[fill-opacity.svg.zip](https://github.com/mchoe/SwiftSVG/files/1729739/fill-opacity.svg.zip)

**Comment by mchoe** [comment]
Thanks for filing the ticket @jbdujardin. This is fixed in v2.1.4, so you can go ahead and update to get this change.

**Comment by jbdujardin** [comment]
Thanks a lot! 👍
